starwars-character-guide/src/CharacterDetails.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class CharacterDetails extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: {},
+            isLoading: true,
+            hasError: false
+        };
+    }
+
+    render() {
+        if (this.state.isLoading) {
+            return <div>
+                <div className="character-details"> Loading... </div>
+                <Link className="link-back" to={'/'} >&lt; Back</Link>    
+            </div>;
+        }
+
+        const character = this.state.data;
+        const charId = this.props.match.params.id;
+        const imageUrl = `https://starwars-visualguide.com/assets/img/characters/${charId}.jpg`
```

Comment:
Looks like these are hosted on Firebase? Nice! Nit: you may want to make the alt tag below the character name.